app/src/main/sentences/fr/open.yml

```diff
@@ -1,2 +1,2 @@
 query:
-  - ouvre<r|z?>|lance<r|z?>|charge<r|z?>|execute<r|z?> (l? app|application)? .what.
+  - ouvrir<?>|ouvre<z?>|lance<r|z?>|charge<r|z?>|execute<r|z?> (l? app|application)? .what.
```

Comment:
`<?>` is not valid, but I guess you wanted to make the `r` optional? So the options would be ouvri, ouvrir, ouvre, ouvrez, is that correct? Also, what about the other words (`lance<r|z?>`, `charge<r|z?>`, `execute<r|z?>`), are these three options valid or is something missing: e.g. lance, lancer, lancez?
Note that if there is an additional incorrect word it's not an issue, but it's important that the correct words are all there.

Reply:
Hello, sorry for the mistake

the word "ouvrer" in french does not exist, it is "ouvrir".

to open = ouvrir

(but "ouvre" and "ouvrez" are valid)

All the others are OK too.

So the correction would be : ouvrir|ouvre<z?>|lance<r|z?> ....

Reply:
Ok, thank you!